amanuensis/resources/fence/__init__.py

```diff
@@ -0,0 +1,53 @@
+
+import requests
+import json
+import os
+
+from amanuensis.auth import sign
+from amanuensis.auth.auth import get_jwt_from_header
+from amanuensis.auth.errors import NoPrivateKeyError
+from cdislogging import get_logger
+
+
+logger = get_logger(__name__)
+
+
+def fence_get_users(usernames, config):
+    '''
+    amanuensis sends a request to fence for a list of user ids 
+    matching the supplied list of user email addresses
+    '''
+    queryBody = {
+        'usernames': usernames
+    }
+
+    # logger.info(f"queryBody usernames: {usernames}")
+
+    try:
+        url = 'http://fence-service/admin/users/selected'  
```

Comment:
Can the hostname of this URL (http://fence-service) be in the config instead of hardcoded. In kubernertes we'll need to inject the actual URL since fence-service is just a docker compose name

Reply:
That's a good idea.  I carried over code from PcdcAnalysisTools to use as an example.  It's hardcoded there as well and PcdcAnalysisTools does not have a config.yaml file in compose-services. Is that something that needs to be corrected, too?

Reply:
yes, you are right, we should update that there as well

Reply:
`PcdcAnalysisTools_settings.py` this is where you can set it. It is in compose-services.template